src/pages/Pricing/PricingPage.js

```diff
@@ -1,13 +1,35 @@
-import React, { Fragment } from 'react';
+import React, { Fragment, useCallback, useEffect, useState, useMemo } from 'react';
+import axios from 'axios';
 import CTA from 'organisms/cta/CTA';
 import PricingFAQ from 'organisms/pricing/PricingFAQ';
 import PricingPlan from 'organisms/pricing/PricingPlan';
 
 const PricingPage = () => {
+  // console.log('princing page component');
+  const [plans, setPlans] = useState([]);
+  const [questions, setQuestions] = useState([]);
+  // const [selectedPlan, setSelectedPlan] = useState('');
+  useEffect(() => {
+    getData();
+  }, []);
+  const getData = async () => {
+    const [{ data: plans }, { data: faqs }] = await Promise.all([
+      axios.get('/api/pricing/plans'),
+      axios.get('/api/pricing/faq')
+    ]);
+    setPlans(plans);
+    setQuestions(faqs);
+  };
+  // get selected plan id
+  const selectedPlanHandler = useCallback((data) => {
+    // setSelectedPlan(data);
+    console.log(data);
```

Comment:
All the console logs have to be removed + the commented out code which is not used.
It is very very rare that we leave the commented out code in the project. 
If we have a task with code that we might need to add back in the project, and it takes time to write it again, we will note a hash of the current git history and add it to the task where it is required to be removed as a comment.

And in the commit a developer has to write a short comment that describes removal of the functionality. Such as "Removed Functionality: blah blah". That was in `git log` it is very easy to find commit where that code was removed in case it has to be put back.

But JIRA or any other task management system must have a comment where that was removed. However when we create tasks FT-41 and those are linked with the task, it is very easy to track exactly where that functionality was removed in case we need to add it back. ;)